content/collections/analytics/en/web-analytics.md

```diff
@@ -39,30 +39,31 @@ If you haven’t already read up on the basics of building charts in Amplitude,
 
 
 ## Analyze industry-standard metrics
-Out-of-the-box Web Analytics offers four sets of insights with default metrics, as well as optional goals or key outcomes, that you are driving users to:
+Out-of-the-box Marketing Analytics offers four sets of insights with default metrics, as well as optional goals or key outcomes, that you are driving users to:
 
-- **Traffic by Channel**: Provides an overview of users visiting your product by channel. Metrics include visitors, bounce rate, session totals, average session duration, sessions per user, and web analytics views.
+- **Traffic by Channel**: Provides an overview of users visiting your product by channel. Metrics include visitors, bounce rate, session totals, average session duration, sessions per user, and marketing analytics views.
 - **Traffic by Campaign**: Provides an overview of the users visiting your product by campaign. Metrics include visitors, bounce rate, session totals, average session duration, sessions per user, and any defined goals.
+- **Ads**: Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, Click-Through Rate (CTR), Cost Per Click (CPC), Customer Acquisition Cost (CAC), and Return on Ad Spend (ROAS), plus flexible settings for acquisition and revenue tracking.
```

Comment:
Is there a way to mark just this section as Beta?

I saw this was used elsewhere:

```
{{partial:admonition type="beta" heading=""}}
This feature is in beta and is in active development. Features and functionality may change during the course of the beta period.
{{/partial:admonition}}
```

but it looks like it's too big -- would apply to an entire section, not just one bullet.